acceptance/indy_acceptance/indy-tests/libraries/result.py

```diff
@@ -0,0 +1,127 @@
+"""
+Created on Nov 9, 2017
+
+@author: nhan.nguyen
+
+Containing classes to make the test result as a json.
+"""
+
+import json
+import time
+import os
+import errno
+from .constant import Colors
+
+
+class KeyWord:
+    TEST_CASE = "testcase"
+    RESULT = "result"
+    START_TIME = "starttime"
+    DURATION = "duration"
+    RUN = "run"
+    STEP = "step"
+    STATUS = "status"
+    MESSAGE = "message"
+
+
+class Status:
```

Comment:
This class must be a subclass of `Enum`.

Reply:
Updated. Please refer to the latest code.